flux-accounting-guide.rst

```diff
@@ -138,6 +138,40 @@ This current job usage is then added to the association's previous job usage
 stored in the flux-accounting database. This sum then represents the
 association's overall job usage.
 
+****************************
+Multi-Factor Priority Plugin
+****************************
+
+The multi-factor priority plugin is a jobtap_ plugin that generates
+an integer job priority for incoming jobs in a Flux system instance. It uses
+a number of factors to calculate a priority and, in the future, can add more
+factors. Each factor has an associated integer weight that determines its
+importance in the overall priority calculation. The current factors present in
+the multi-factor priority plugin are:
+
+* **fair-share**: the ratio between the amount of resources allocated vs. resources
+  consumed.
+
+* **urgency**: a user-controlled factor to prioritize their own jobs.
+
+In addition to generating an integer priority for submitted jobs in a Flux
+system instance, the multi-factor priority plugin also enforces per-association
+job limits to regulate use of the system. The two per-association limits
+enforced by this plugin are:
+
+* **max_active_jobs**: a limit on how many _active_ jobs an association can have at
+  any given time. Jobs submitted after this limit has been hit will be rejected
+  with a message saying that the association has hit their active jobs limit.
+
+* **max_running_jobs**: a limit on how many _running_ jobs an association can have
+  at any given time. Jobs submitted after this limit has been hit will be held
+  until one of the association's currently running jobs finishes running.
```

Comment:
Perhaps note here that jobs "will be held by adding a `max-running-jobs` dependency" or similar.

Reply:
Oh, that's a good suggestion. I'll go ahead and add this to the description, Thanks!